packages/dds/tree/src/index.ts

```diff
@@ -159,6 +159,23 @@ export {
 	getJsonSchema,
 	type LazyItem,
 	type Unenforced,
+	type SimpleNodeSchemaBase,
+	type SimpleTreeSchema,
+	type SimpleNodeSchema,
+	type SimpleFieldSchema,
+	type SimpleLeafNodeSchema,
+	type SimpleMapNodeSchema,
+	type SimpleArrayNodeSchema,
+	type SimpleObjectNodeSchema,
+	normalizeFieldSchema,
+	isTreeNodeSchemaClass,
+	normalizeAllowedTypes,
```

Comment:
I think we should avoid exporting these 3 if we can. The 2 normalization functions in particular shouldn't be needed externally (our APIs should do this automatically under the hood).

I'm happy to look at any cases where these are needed to see if there are viable alternatives.

`isTreeNodeSchemaClass` also really shouldn't be an external concern - that's used internally to differentiate between 2 internal implementation details.